### PR TITLE
Update uswds

### PR DESCRIPTION
**Pull request by slj**
Connects https://github.com/department-of-veterans-affairs/caseflow-commons/issues/23

- Rails was throwing a "Missing 'component' of type/javascript" error because the **components.js** file was missing (this was already in the old gem we were using) - adding it to `vendor/assets/javascripts` took care of that.
- After updating file paths the tests broke, it turned out that the input fields were too close to the labels; moving them out the way via CSS solved that.
- Some tests were modified to make `trigger('click')` unnecessary.

### CODE REVIEW

File: vendor/assets/javascripts/components.js [hunk 1]
```diff
@@ -0,0 +1,376 @@
+'use strict';
```
Comment (shanear) [inline vendor/assets/javascripts/components.js]: Why are we having to add this file?

It seems like these javascripts should be able to be loaded through a Gem or NPM package.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -90,3 +90,7 @@
   #   # as the one that triggered the failure.
   #   Kernel.srand config.seed
 end
+
+def find_label_for(val)
+  find(:xpath, "//label[@for='#{val}']")
```
Comment (shanear) [inline spec/spec_helper.rb]: Awesome helper function! 💯

Comment (slj) [inline spec/spec_helper.rb]: Thx!

File: vendor/assets/javascripts/components.js (hunk 1 above)
Comment (slj) [inline vendor/assets/javascripts/components.js]: It was part of the harris gem that @mmlumba was leveraging (https://github.com/harrisj/us_web_design_standards_gem/blob/master/app/assets/javascript/components.js).  I don't think this is a permanent solution, but it's the easiest path forward for the moment.

Comment (mmlumba) [inline vendor/assets/javascripts/components.js]: This file `components.js` was what I was using for the accordions on the style guide, just to provide context.

Comment (shanear) [inline vendor/assets/javascripts/components.js]: Does the file not exist on the new Gem?

Comment (mmlumba) [inline vendor/assets/javascripts/components.js]: From the readme of the new gem: At the moment, we've confirmed that the fonts and stylesheets in this repo work in Rails apps. JavaScript is TBD.

Comment (shanear) [inline vendor/assets/javascripts/components.js]: ugh, okay, can we make a tech-improvement issue to remove this file and find a better way to deal with it?

Comment (slj) [inline vendor/assets/javascripts/components.js]: Done: https://github.com/department-of-veterans-affairs/caseflow-commons/issues/57

**Pull request merged**